minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -28,6 +28,16 @@ conf.dir=./conf
 # How long to wait after telling MiNiFi to shutdown before explicitly killing the Process
 graceful.shutdown.seconds=20
 
+# The location for the configuration file
+nifi.minifi.config=./conf/config.yml
+
+# Notifiers to use for the associated agent, comma separated list of class names
+#nifi.minifi.notifier.components=org.apache.nifi.minifi.bootstrap.configuration.FileChangeNotifier
+
+# File change notifier configuration for path of the file to monitor and the associated polling period
+#nifi.minifi.notifier.file.config.path=
```

Comment:
I agree that there may be use-cases where there will be an separate directory to watch but as a user that has access to the MINIFI directory directly, my natural thinking is to modify the file itself. If it's not too much trouble to optimize now I would lean towards that though if it would be a lot of needless work we can just properly document the limitations.